net/ip_test.go

```diff
@@ -674,3 +674,24 @@ func TestMaskLastNBits(t *testing.T) {
 		assert.Equal(t, test.expected, res, test.name)
 	}
 }
+
+func TestIPv4FromBytes(t *testing.T) {
+	resultZero := IPv4FromBytes([]byte{})
```

Comment:
a more idomatic way to write this test would have been to use a table test like in `TestMaskLastNBits`. Since the testcases are very stable I will merge this anyways. No need to overengineer the tests.